src/libs/actions/Session.js

```diff
@@ -234,6 +238,35 @@ function signIn(password, twoFactorAuthCode) {
         });
 }
 
+/**
+ * Uses a short lived authToken to continue a user's session from OldDot
+ *
+ * @param {String} accountID
+ * @param {String} email
+ * @param {String} shortLivedToken
+ * @param {string} encryptedAuthToken
+ */
+function signInWithShortLivedToken(accountID, email, shortLivedToken, encryptedAuthToken) {
+    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, loading: true});
+
+    createTemporaryLogin(shortLivedToken, encryptedAuthToken, email).then((response) => {
+        Onyx.merge(ONYXKEYS.SESSION, {
+            authToken: shortLivedToken,
+            accountID,
+            email,
+        });
+        if (response.jsonCode === 200) {
+            getUserDetails();
```

Comment:
Mirroring comment from @marcaaron, but do we need this? And if we do, shouldn't we also include these other two:
https://github.com/Expensify/App/blob/a3cf706b5ec1b17f1290b3f4ed758c7c6498a94e/src/libs/Navigation/AppNavigator/AuthScreens.js#L186-L187

Reply:
Since we're transitioning to an AuthScreen before doing the exitTo though, I think it's fine if we don't include `getUserDetails`

Reply:
I have no idea

Reply:
Maybe it was added to support a case where we are already logged in as someone else. I don't think we need to worry about that and can probably test that flow in a follow up / remove this code for now.